client/account-recovery/controller.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import DocumentHead from 'components/data/document-head';
+import LostPassword from 'account-recovery/lost-password';
+
+const ANALYTICS_PAGE_TITLE = 'Account Recovery';
+
+export default {
+	lostPassword( context ) {
+		const basePath = context.path;
+
+		analytics.pageView.record( basePath, `${ ANALYTICS_PAGE_TITLE } > Lost Password` );
+
+		renderWithReduxStore(
```

Comment:
To my previous point about using the isomorphic router, an advantage is not having to explicitly render into the DOM element:

https://github.com/Automattic/wp-calypso/blob/727fc973a841f28ff08dea300aa19813b518eec5/client/my-sites/types/controller.jsx#L18-L24